### DISCUSSION TITLE
How to generate SSH key title on my github

### DISCUSSION THREAD

**Discussion by JoyTeck** [community/community#122057] [New to GitHub]
How do I generate a new SSH key? I tried creating different but keeps saying I should create one in OpenSSH form

**Comment by davevad93** [comment]
Hi @JoyTeck ,

I'll leave you a couple of links that you might find useful:

1. This is the official GitHub documentation about [Generating a new SSH key and adding it to the ssh-agent](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
2.  If the first link isn't enough to solve your problem, here's another one from an old discussion of the Community. [Key is invalid. you must supply a key in openssh public key format](https://github.com/orgs/community/discussions/23089)

**Comment by JoyTeck** [comment]
Thank you Sent from Mail for Windows From: Davide PrestiSent: Thursday, May 2, 2024 10:56 PMTo: community/communityCc: JoyTeck; MentionSubject: Re: [community/community] How to generate SSH key title on my github (Discussion #122057) Hi @JoyTeck ,I'll leave you a couple of links that you might find useful:This is the official GitHub documentation about Generating a new SSH key and adding it to the ssh-agentIf the first link isn't enough to solve your problem, here's another one from an old discussion of the Community. Key is invalid. you must supply a key in openssh public key format—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Reply by davevad93** [reply]
You're welcome @JoyTeck , if you have solved your problem and believe I answered your question properly, please, mark it as `answered`.
have a nice day!

**Comment by jamesleeksc** [comment]
In git bash
ssh-keygen -t rsa -C "your email" -f "your name"
